services/121-service/src/payments/fsp-integration/intersolve-visa/mappers/intersolve-visa-status.mapper.ts

```diff
@@ -115,15 +101,15 @@ export class IntersolveVisaStatusMappingService {
     }
   }
 
-  private getLinks(
+  private static getLinks(
     actions: string,
     linkCreationInfo: VisaCardLinkCreationInfo,
-  ): VisaCardActionLink[] {
+  ): VisaCardActionLinkDto[] {
     if (!actions) {
       return [];
     }
 
-    const links: VisaCardActionLink[] = [];
+    const links: VisaCardActionLinkDto[] = [];
```

Comment:
So in our old code we used to have links according to https://en.wikipedia.org/wiki/HATEOAS, but this functionality is
-  currently not really used by our frontend
- we never use this HATEOAS somewhere else
- complicates these mapper functions because programId and referenceId need to be passes down through all functions
So it does not add a lot of value in my opinion.

Reply:
@RubenGeo 
These hyperlinks SHOULD have been used by our front-end :)
But if they are not, then how does the front-end currently decide which actions to show?

Reply:
![image](https://github.com/global-121/121-platform/assets/34537157/1624fa4b-d772-4280-8606-9a56b6130520)
It only uses the actions part and not the href or method. The href and method are basically stored here where we store all our api requests interfaces/Portal/src/app/services/programs-service-api.service.ts